_includes/header.html

```diff
@@ -17,12 +17,12 @@
         <ul class="nav navbar-nav">
             <li><a href="/news/"><i class="fa fa-commenting-o"></i> News</a></li>
             <li><a href="/download/"><i class="fa fa-download"></i> Download</a></li>
-            <li><a href="/documentation/"><i class="fa fa-book"></i> Documentation</a></li>
+            <li><a href="https://docs.nunit.org" target="_blank"><i class="fa fa-book"></i> Documentation</a></li>
```

Comment:
I think we can merge this without a needing Rob’s explicit approval. 😊 Do we definitely want the target=blank? I would have thought docs should be a more natural part of the site, rather than something like Twitter - which is a new tab thing.

No strong opinions however. 😊

Reply:
@ChrisMaddock I don't have a strong opinion either way. I tend to use a `blank` target when I know the look & feel of the site will be different, so that it's clear it's a different entity entirely. I've got no issues if we'd prefer to remove that. 👍

Reply:
I'd go by the availability of the top level menu. If it's not there then open in a new tab.

Reply:
That seems a sensible convention. 🙂